src/test/java/org/jenkins/plugins/lockableresources/LockStepTest.java

```diff
@@ -516,6 +519,48 @@ public void evaluate() throws Throwable {
 		});
 	}
 
+	@Issue("JENKINS-40368")
+	@Test
```

Comment:
I think you want to `@Ignore` this so that

* this could be merged at any time
* #46 can merge this branch and delete the ignore.